tests/test_oauth.py

```diff
@@ -217,7 +217,7 @@ def test_start_flow(self):
         with self.app.app.test_request_context('/oauth/connect/mock1a/'):
```

Comment:
Note: is this why we don't need to worry about session/context issue? (Since `with` destroys the context after it's done.)